api/api/main.py

```diff
@@ -131,6 +133,7 @@ async def lifespan(app: FastAPI):
 
 app.include_router(probes.router)
 app.include_router(run.router)
+app.include_router(webhooks.router)
```

Comment:
Careful. Use the same pattern as the clerk and stripe webhooks in `main_router.py` otherwise the endpoint will be included in the run endpoint

Reply:
https://github.com/WorkflowAI/WorkflowAI/pull/176/commits/cab0983ae3820fde93c46e1d15073b9924c93eb7